addons/survey/views/survey_survey_views.xml

```diff
@@ -65,7 +65,8 @@
                     </group>
                     <notebook>
                         <page string="Questions" name="questions">
-                            <field name="question_and_page_ids" nolabel="1" widget="question_page_one2many" mode="tree,kanban" context="{'default_survey_id': active_id, 'default_questions_selection': questions_selection}">
+                            <field name="question_and_page_ids" nolabel="1" widget="question_page_one2many" mode="tree,kanban" class="d-block"
```

Comment:
So it does make sense to add this here instead of a more global place?

Reply:
Oh you're right! :open_mouth:  Just realized the bug also happen in website_slides :sweat_smile: Will look for a more global solution